backend/de.metas.dunning/src/main/java/de/metas/dunning/api/impl/DefaultDunningProducer.java

```diff
@@ -164,6 +176,22 @@ protected I_C_DunningDoc createDunningDoc(final I_C_Dunning_Candidate candidate)
 		return doc;
 	}
 
+	@Nullable
+	private String getPOReferenceToUse(final I_C_Dunning_Candidate candidate)
```

Comment:
is this code automated tested?

Reply:
does not matter, direct or indirect, junit or cucumber

Reply:
added now DefaultDunningProducerPOReferenceTest.